src/modules/petrology/FeatureDetail.js

```diff
@@ -0,0 +1,269 @@
+          innerRef={featureForm}
+          validateOnChange={true}
+          enableReinitialize={true}
+        >
+          {() => (
+            <View>
+              <ListItem containerStyle={commonStyles.listItemMinimalVerticalPadding}>
+                <ListItem.Content>
+                  <Field
+                    component={TextInputField}
+                    name={'label'}
+                    label={'Label'}
+                    key={'label'}
+                  />
+                </ListItem.Content>
+              </ListItem>
+              <FlatListItemSeparator/>
+              <ListItem containerStyle={commonStyles.listItemMinimalVerticalPadding}>
+                <ListItem.Content>
+                  <Field
+                    component={TextInputField}
+                    name={'name'}
+                    label={'Name'}
+                    key={'name'}
+                  />
+                </ListItem.Content>
+              </ListItem>
+              <FlatListItemSeparator/>
+              <ListItem containerStyle={commonStyles.listItemMinimalVerticalPadding}>
+                <ListItem.Content>
+                  <Field
+                    component={TextInputField}
+                    name={'description'}
+                    label={'Feature Description'}
+                    key={'description'}
+                  />
+                </ListItem.Content>
+              </ListItem>
+              <ListItem containerStyle={commonStyles.listItemMinimalVerticalPadding}>
+                <ListItem.Title style={formStyles.fieldLabel}>{'Feature Type'}</ListItem.Title>
+              </ListItem>
+              <MultiSelect
+                label={'Type'}
+                hideSubmitButton={true}
+                hideTags={false}
+                single={true}
+                styleMainWrapper={formStyles.dropdownSelectedContainer}
+                items={props.featureTypes}
+                uniqueKey={'id'}
+                displayKey={'name'}
+                selectedItems={typeEnum}
+                selectText={type === '' ? '--Select Feature Type--' : '   ' + type}
+                onSelectedItemsChange={fieldValueChanged}
+                fontSize={themes.PRIMARY_TEXT_SIZE}
+                selectedItemTextColor={themes.BLACK}
+                selectedItemIconColor={themes.BLACK}
+                textColor={themes.BLACK}
+                itemTextColor={themes.BLACK}
+                styleRowList={formStyles.fieldValue}
+                styleDropdownMenuSubsection={formStyles.dropdownSelectedContainer}
+              />
+              <FlatListItemSeparator/>
+              {type === 'other' && <ListItem containerStyle={commonStyles.listItemMinimalVerticalPadding}>
+                {type === 'other'
+                && <ListItem.Title style={formStyles.fieldLabel}>{'Other Feature Type'}</ListItem.Title>}
+                <ListItem.Content>
+                  {type === 'other' && (<TextInput
```

Comment:
`<TextInput` should be lined up with its closing tag so drop it down to the next line.